analysis/viewshed-location/src/main/java/com/esri/samples/viewshed_location/ViewshedLocationController.java

```diff
@@ -81,6 +76,11 @@ public void initialize() {
     LocationViewshed viewshed = new LocationViewshed(location, headingSlider.getValue(), pitchSlider.getValue(),
         horizontalAngleSlider.getValue(), verticalAngleSlider.getValue(), minDistanceSlider.getValue(),
         maxDistanceSlider.getValue());
+    // set the colors of the visible and obstructed areas
+    Viewshed.setVisibleColor(0xCC00FF00);
+    Viewshed.setObstructedColor(0xCCFF0000);
+    // set the color of the frustum outline
+    Viewshed.setFrustumOutlineColor(0xCC0000FF);
```

Comment:
Good solution to the issue, to just go ahead and simplify it 👍 
We only need to update the screenshot to use the colours that we're setting here.